### DISCUSSION TITLE
How would i select a specific group on a specific side using lua?

### DISCUSSION THREAD

**Discussion by Ajaymamtora** [lucobellic/edgy-group.nvim#27] [Q&A] [answered]
I want to reveal neo-tree any maybe other stuff in the same group via lua.

Something like `require('edgy-group.stl').view('left', 'g') -- where g is the pick_key

Is something like this possible?

**Comment by Ajaymamtora** [comment]
Maybe open_groups_by_key -- but for a specific side?

**Reply by lucobellic** [reply]
Currently the only identifier for a group is the pick key (but could be duplicated), I will add an option to filter by side, probably next week.

You can still use the index of the group with `require('edgy-group').open_group_index(position, index)`.

**Reply by Ajaymamtora** [reply]
Thanks I'll wait for the update :)

**Reply by lucobellic** [reply, marked as answer]
Added optional parameters as a table in #30. 
Example:
```lua
require('edgy-group').open_groups_by_key('e', { position = 'left' })
```

> [!IMPORTANT]
> The `toggle` option has been moved inside the table

**Reply by Ajaymamtora** [reply]
Thanks that works great
